src/plugins/tables/tables.js

```diff
@@ -434,8 +434,12 @@ $document.on( "submit", ".wb-tables-filter", function( event ) {
 
 			// Verifies if regex was preset, if not preset use 'contains value' as default
 			if ( !$regex ) {
-				$value = $value.replace( /\s/g, "\\s*" );
-				$regex = "(" + $value + ")";
+				if ( $elm[ 0 ].getAttribute( "data-exact" ) === "true" ) {
```

Comment:
Considering the value set on the data attribute don't really matter here because it is simply a flag, let's use[ `hasAttribute( "data-exact" )`](https://dom.spec.whatwg.org/#ref-for-idl-boolean%E2%91%A3%E2%91%A1) instead. If later, the value is somehow use, then it will make senses to check what value the author has specified.

```suggestion
				if ( $elm[ 0 ].hasAttribute( "data-exact" ) ) {
```